packages/interface/src/components/interface-skeleton/style.scss

```diff
@@ -79,6 +79,10 @@ html.interface-interface-skeleton__html-container {
 
 }
 
+.interface-interface-skeleton__drawer {
+	position: relative;
```

Comment:
Why is this needed?

Reply:
Ah, it was needed because of the preview `left` value, but I've updated it now, and this positioning is not needed anymore.